section2-the-ruby-programming-language/methods/methods.rb

```diff
@@ -0,0 +1,62 @@
+# define methods for arithmetic calculations
+def add(first_num, second_num)
+    first_num.to_f + second_num.to_f
+end
+
+def subtract(first_num, second_num)
+    first_num.to_f - second_num.to_f
+end
+
+def multiply(first_num, second_num)
+    first_num.to_f * second_num.to_f
+end
+
+def divide(first_num, second_num)
+    first_num.to_f / second_num.to_f
```

Comment:
question: why do you need to do `.to_f`?

Reply:
In order for arithmetic operations to work, we need it to be an integer or float. `get.chomp` gets input as string object so by using `.to_f`, we change the string object to float when using division. We choose `.to_f` over `.to_i` because if using integer for both numbers will result in an integer division. 

In integer division and modulus, the dividend is divided by the divisor into an integer quotient and a remainder. The integer quotient operation is referred to as integer division, and the integer remainder operation is the modulus. In the real world of data manipulation there are some things that are always handled in whole units or numbers (ex. integer data type). Thus when using two integers it results in an integer division.

As we want our division for example:
`10` divided by `4` to be `2.5`
and not `2`

Reply:
Ok.

Question: why does 10/4 = 2 in ruby?

Reply:
Because two integers will result in an integer division in ruby.
If we want to have a float division, we can change either one of the numbers to float or both the numbers to float:
10/4.0 or 10.0/4.0

Reply:
ok. Why does 2 integers will result in an integer division in ruby?

Reply:
In Ruby, the division operator depends on the class of the operand: if both operands are integers, then truncating integer division is performed thus resulting an integer. If either operand is a float, then floating-point division is performed.

Reply:
Speak english please